### PR TITLE
OW-342

### PR DESCRIPTION
**Pull request by gmininberg**

### CODE REVIEW

**Comment by joliveraortega** [comment]
There is a typo in the name of the new folder (`examples/mediapipe/BackgroundEnchantments`) for the sample being added.

File: examples/mediapipe/BackgroundEnchantments/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Background Enhancement Sample Application
```
Comment (joliveraortega) [inline examples/mediapipe/BackgroundEnchantments/README.md:1]:
```suggestion
# Background Effects Sample Application
```

File: examples/mediapipe/BackgroundEnchantments/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+This sample application shows how to create background enhancement using [@vonage/ml-transformers](https://www.npmjs.com/package/@vonage/ml-transformers) package.
```
Comment (joliveraortega) [inline examples/mediapipe/BackgroundEnchantments/README.md:2]:
```suggestion
his sample application shows the result of applying some background visual effects by using the [@vonage/ml-transformers](https://www.npmjs.com/package/@vonage/ml-transformers) library.
```

File: examples/mediapipe/BackgroundEnchantments/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+# Demo
+You can see a demo of this sample running [here](https://vonage-background-enchantments-sample.s3.amazonaws.com/index.html)
```
Comment (joliveraortega) [inline examples/mediapipe/BackgroundEnchantments/README.md:4]:
```suggestion
You can see a live demo of this sample running [here](https://vonage-background-enchantments-sample.s3.amazonaws.com/index.html)
```

File: examples/mediapipe/BackgroundEnchantments/README.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+# Building the application
+**This is very simple**
```
Comment (joliveraortega) [inline examples/mediapipe/BackgroundEnchantments/README.md:6]: IMO this not add any info the build process so I would remove it.

File: examples/mediapipe/BackgroundEnchantments/README.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+## Run dev version
+
+    npm run dev
+
+## Run production version
+
+    npm run build
+    npm run preview
+
+# Using it in the "real world"
```
Comment (joliveraortega) [inline examples/mediapipe/BackgroundEnchantments/README.md:20]:
```suggestion
# Using the library in a real use case
```

File: examples/mediapipe/BackgroundEnchantments/README.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+In our example the result is shown on side canvas.
+However, in the real world you want to use it with [OT.Publisher](https://tokbox.com/developer/guides/vonage-media-processor/js/#publisher-setvideomediaprocessorconnector-method)
+**That is simple too**
```
Comment (joliveraortega) [inline examples/mediapipe/BackgroundEnchantments/README.md:23]: These kind of sentences do not add any value to how the integration is or the library is used in a real use case. I would remove it.

File: examples/mediapipe/BackgroundEnchantments/README.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+    npm run dev
+
+## Run production version
+
+    npm run build
+    npm run preview
+
+# Using it in the "real world"
+In our example the result is shown on side canvas.
```
Comment (joliveraortega) [inline examples/mediapipe/BackgroundEnchantments/README.md:21]:
```suggestion
This sample apps renders the result of the video captured from the camera on the right side canvas.
```

File: examples/mediapipe/BackgroundEnchantments/README.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+However, in the real world you want to use it with [OT.Publisher](https://tokbox.com/developer/guides/vonage-media-processor/js/#publisher-setvideomediaprocessorconnector-method)
```
Comment (joliveraortega) [inline examples/mediapipe/BackgroundEnchantments/README.md:22]:
```suggestion
Some real use case for this library would be using it along with [OT.Publisher](https://tokbox.com/developer/guides/vonage-media-processor/js/#publisher-setvideomediaprocessorconnector-method).
```

File: examples/mediapipe/BackgroundEnchantments/index.html [hunk 9]
```diff
@@ -0,0 +1,132 @@
...
+  <title>MediaPipe sample</title>
+  <link rel="icon" href="https://vonage-client-media-samples.s3.amazonaws.com/vonage.png">
+</head>
+
+<body>
+  <vwc-top-app-bar alternate>
+    <img slot="navigationIcon" height="24"
+      src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQQI9k8NKq3KoOs1VkwSYwPwwQbisiS_K-0jtBUyml13k7EBy8Aekz7G7I1vqceWf5kEQ&usqp=CAU" />
+    <div>
+      <vwc-banner open connotation="info" message="Vonage Background Enchantments (out of the box soloution) sample"></vwc-banner>
```
Comment (joliveraortega) [inline examples/mediapipe/BackgroundEnchantments/index.html:70]: Typo.

File: examples/mediapipe/BackgroundEnchantments/README.md (hunk 1 above)
Comment (gmininberg) [reply examples/mediapipe/BackgroundEnchantments/README.md:1]: fixed

File: examples/mediapipe/BackgroundEnchantments/README.md (hunk 2 above)
Comment (gmininberg) [reply examples/mediapipe/BackgroundEnchantments/README.md:2]: fixed

File: examples/mediapipe/BackgroundEnchantments/index.html [hunk 10]
```diff
@@ -0,0 +1,132 @@
...
+    import "@vonage/vwc-banner";
+    import "@vonage/vwc-card";
+    import "@vonage/vwc-switch";
+    import "@vonage/vwc-select";
+    import "@vonage/vwc-list/vwc-list-item";
+    import "@vonage/vwc-snackbar";
+  </script>
+
+  <title>MediaPipe sample</title>
+  <link rel="icon" href="https://vonage-client-media-samples.s3.amazonaws.com/vonage.png">
```
Comment (joliveraortega) [inline examples/mediapipe/BackgroundEnchantments/index.html:62]: Is this URL correct?

File: examples/mediapipe/BackgroundEnchantments/index.html [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+</head>
+
+<body>
+  <vwc-top-app-bar alternate>
+    <img slot="navigationIcon" height="24"
+      src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQQI9k8NKq3KoOs1VkwSYwPwwQbisiS_K-0jtBUyml13k7EBy8Aekz7G7I1vqceWf5kEQ&usqp=CAU" />
+    <div>
+      <vwc-banner open connotation="info" message="Vonage Background Enchantments (out of the box soloution) sample"></vwc-banner>
+      <vwc-layout gutters="xs" column-basis="block">
+        <vwc-text font-face="subtitle-1" class="headline" tight>This sample application shows how to use Vonage Background Enchantments soloution.</vwc-text>
```
Comment (joliveraortega) [inline examples/mediapipe/BackgroundEnchantments/index.html:72]: Typos.

File: examples/mediapipe/BackgroundEnchantments/README.md (hunk 3 above)
Comment (gmininberg) [reply examples/mediapipe/BackgroundEnchantments/README.md:4]: fixed

File: examples/mediapipe/BackgroundEnchantments/index.html [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+                </vwc-list-item>
+                <vwc-list-item value="video">
+                  Video
+                </vwc-list-item>
+                <vwc-list-item value="silhouettLow">
+                  Silhouette Low
+                </vwc-list-item>
+                <vwc-list-item value="silhouetteHigh">
+                  Silhouette High
+                </vwc-list-item>
```
Comment (joliveraortega) [inline examples/mediapipe/BackgroundEnchantments/index.html:107]: Typos above.

File: examples/mediapipe/BackgroundEnchantments/main.ts [hunk 13]
```diff
@@ -0,0 +1,138 @@
...
+    alert('Something bad happened: ' + e);
+  }
+
+  const cameraswitchSelector: any = document.getElementById('cameraswitch')
+  const githubButtonSelector: HTMLElement | null = document.getElementById("githubButton")
+  const vividButtonSelector: HTMLElement | null = document.getElementById("vividButton")
+  const backgroundPostProcessSelector: any = document.getElementById('postProcessSelector')
+  let cameraSwitchChecked: boolean = false
+
+  const MEDIA_ASSETS_URI: string = "https://vonage-background-enchantments-sample.s3.amazonaws.com/"
```
Comment (joliveraortega) [inline examples/mediapipe/BackgroundEnchantments/main.ts:18]: Typo in URL name.

File: examples/mediapipe/BackgroundEnchantments/README.md (hunk 5 above)
Comment (gmininberg) [reply examples/mediapipe/BackgroundEnchantments/README.md:20]: fixed

File: examples/mediapipe/BackgroundEnchantments/README.md (hunk 4 above)
Comment (gmininberg) [reply examples/mediapipe/BackgroundEnchantments/README.md:6]: fixed

File: examples/mediapipe/BackgroundEnchantments/index.html (hunk 11 above)
Comment (joliveraortega) [reply examples/mediapipe/BackgroundEnchantments/index.html:72]: Effects maybe?

Comment (joliveraortega) [inline examples/mediapipe/BackgroundEnchantments/index.html]:
```suggestion
        <vwc-text font-face="subtitle-1" class="headline" tight>This sample application shows the result of applying some background visual effects.</vwc-text>
```

**Review by joliveraortega — APPROVED** [review]

**Pull request merged**